app/Http/ViewComposers/Admin/LeftMenuComposer.php

```diff
@@ -16,6 +16,13 @@ public function compose(View $view)
                 'icon' => 'fa fa-th',
                 'active' => 'index-admin',
             ],
+
+            [
```

Comment:
This is hardcoded logic and not checks if static-pages package is installed.
So you have to create some private method that iterates package-names and if they are installed, adds them to menu.